Makefile

```diff
@@ -6,18 +6,18 @@ help:
 	@echo "Available commands:"
 	@grep '^[^#[:space:]].*:' Makefile | grep -v '^default' | grep -v '^_' | sed 's/://' | xargs -n 1 echo ' -'
 
-coverage:
-	rm -rf coverage; ./vendor/bin/phpunit --coverage-html=coverage/ --coverage-clover=coverage/clover.xml
+fix-code-standards:
+	./vendor/bin/php-cs-fixer fix --verbose
 
-all-tests:
-	$(MAKE) unit-tests
-	$(MAKE) integration-tests
+test:
+	$(MAKE) test-integration
+	$(MAKE) test-unit
```

Comment:
I would do the unit tests first, cozz they are faster and more fine grained, so its easier to pin-point problems with them then integration tests. (if the test suite breaks, the 2nd test suite will not run)